tests/Unit/Core/Cache/ModuleTemplateCacheWarmerTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * 2007-2019 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace Tests\Unit\Core\Cache;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShopBundle\Cache\ModuleTemplateCacheWarmer;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Twig\Environment;
+use Twig\Loader\ArrayLoader;
+
+// REVIEW ME: Is this class well located?
```

Comment:
Sorry for the late answer 😞 

Usually we mirror the class location / namespace for its unit test, so it would locate this test in namespace `PrestaShopBundle/Cache/` rather. Do you wish to drop the PrestaShopBundle for Core ?